src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
You need to create a _reset.scss in utils folder to reset css.

Reply:
Not necessary, this project is small enough to put it here.